### ISSUE TITLE
Upgrade PatternFly, remove workarounds for fixed bugs, fix regressions

### ISSUE THREAD

**Issue by mturley** [konveyor/forklift-ui#540]
I attempted to upgrade PatternFly in https://github.com/konveyor/forklift-ui/pull/508 and ran into several issues that were not worth addressing before the GA code freeze. We should take care of these sooner rather than later when working towards 2.1.0.

This includes https://github.com/konveyor/forklift-ui/issues/210, which may be worth addressing as its own PR, but this issue should not be closed until that one is.

This will also unblock https://github.com/konveyor/forklift-ui/issues/463.
